meta-openpli/recipes-openpli/streamproxy/streamproxy.bb

```diff
@@ -11,6 +11,8 @@ PKGV = "2+git${GITPKGV}"
 RDEPENDS_${PN} = "enigma2-plugin-systemplugins-transcodingsetup"
 
 SRC_URI = "git://github.com/eriksl/streamproxy.git;protocol=git"
+SRC_URI_append_libc-musl = " file://0001-streamproxy-fix-build-with-musl.patch"
```

Comment:
@andrea-adami Would you please send a PR for https://github.com/eriksl/streamproxy or @eriksl could apply this patch.